### DISCUSSION TITLE
Public ports stopped working

### DISCUSSION THREAD

**Discussion by treeder** [community/community#6091] [Codespaces] [answered]
Seems they are all going through the auth cycle (ie: same as when it's private) or I'm getting a gateway timeout 504.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @treeder, thanks for reaching out. Can you please share the following with me via email (tanmayeekamath@github.com):
- Your codespace id.
- Are you using the codespaces in the browser or VS Code Desktop? 
- [Codespace logs ](https://docs.github.com/en/codespaces/troubleshooting/codespaces-logs)

We'll investigate and get back to you shortly.

**Reply by derekbekoe** [reply]
We’ve identified the issue and a fix is in progress.
It should be released by tomorrow.

**Reply by derekbekoe** [reply]
Until the fix is deployed, you can switch your Codespace to VS Code Insiders and then make the port public.

![image](https://user-images.githubusercontent.com/16448634/136868108-22ab7924-9bb3-47fc-ba98-3ec4b020bc3e.png)

**Reply by treeder** [reply]
Great, thanks.

**Reply by derekbekoe** [reply]
This change has been deployed.
